src/d_main.c

```diff
@@ -1633,16 +1633,21 @@ static boolean AllowEndDoom(void)
           || exit_sequence == EXIT_SEQUENCE_ENDOOM_ONLY));
 }
 
-static boolean AllowEndDoomPWADOnly() {
-  return (!W_IsIWADLump(W_CheckNumForName("ENDOOM")) && endoom_pwad_only);
-}
+
 
 static void D_EndDoom(void)
 {
-  if (AllowEndDoom() && AllowEndDoomPWADOnly())
-  {
-    D_ShowEndDoom();
-  }
+  // Do we even want to show an ENDOOM?
+  if (!AllowEndDoom()) return;
```

Comment:
Please move the `return` to the next line, indented. Preferably surrounded by braces on separate lines.